src/lib.rs

```diff
@@ -885,51 +925,747 @@ impl GooseAttack {
+                        detail: "The GooseDefault::ThrottleRequests default can not be set to more than 1,000,000 requests per second.".to_string(),
+                    });
+                }
+                throttle_requests
+            }
+        } else {
+            0
+        };
+
+        if self.throttle_requests > 0 {
+            info!("throttle_requests = {}", self.throttle_requests);
+        }
+
+        Ok(())
+    }
+
+    // Determine if no_reset_statics is enabled.
+    fn no_reset_metrics(&self) -> bool {
+        if self.configuration.no_reset_metrics {
+            true
+        } else if let Some(default) = self.defaults.no_reset_metrics {
+            // Do not default to no_reset_metrics on Worker.
+            if self.attack_mode == GooseMode::Worker {
+                false
+            } else {
+                default
+            }
+        } else {
+            false
+        }
+    }
+
+    // Determine if the status_codes flag is enabled.
+    fn set_status_codes(&mut self) {
+        // Overload self.configuration.status_codes so it's available on Worker.
+        if !self.configuration.status_codes {
+            self.configuration.status_codes = if let Some(default) = self.defaults.status_codes {
+                // Do not default to status_codes on Worker.
+                if self.attack_mode == GooseMode::Worker {
+                    false
+                } else {
+                    default
+                }
+            } else {
+                false
+            };
+        }
+    }
+
+    // Determine if the only_summary flag is enabled.
+    fn set_only_summary(&mut self) {
+        // Overload self.configuration.only_summary so it's available on Worker.
+        if !self.configuration.only_summary {
+            self.configuration.only_summary = if let Some(default) = self.defaults.only_summary {
+                // Do not default to only_summary on Worker.
+                if self.attack_mode == GooseMode::Worker {
+                    false
+                } else {
+                    default
+                }
+            } else {
+                false
+            };
+        }
+    }
+
+    // Determine if the no_task_metrics flag is enabled.
+    fn set_no_task_metrics(&mut self) {
+        // Overload self.configuration.no_task_metrics so it's available to Users.
+        if !self.configuration.no_task_metrics {
+            self.configuration.no_task_metrics =
+                if let Some(default) = self.defaults.no_task_metrics {
+                    // Do not default to no_task_metrics on Worker.
+                    if self.attack_mode == GooseMode::Worker {
+                        false
+                    } else {
+                        default
+                    }
+                } else {
+                    false
+                };
+        }
+    }
+
+    // Determine if the no_metrics flag is enabled.
+    fn set_no_metrics(&mut self) {
```

Comment:
@LionsAd Again we overload `configuration`, in this case so it's available to `GooseUser`'s